library/objective-c/EnvoyEngineImpl.m

```diff
@@ -261,6 +268,17 @@ - (int)registerFilterFactory:(EnvoyHTTPFilterFactory *)filterFactory {
   return kEnvoySuccess;
 }
 
+- (int)registerStringAccessor:(NSString *)name
+               stringAccessor:(EnvoyStringAccessor *)stringAccessor {
```

Comment:
Have we considered registering a lambda/closure that returns a string instead? That'll probably be easier to interact with on the platform layer, since `getString()` is a little overly generic IMO